### PR TITLE
Adding the Table of contents

### LINKED ISSUE #666: Create a Table of Contents in the Readme
### What do you think about adding Table of Contents to readme file since it is very long and hard to navigate through😃

_Originally posted by @puneeth072003 in https://github.com/OWASP/wrongsecrets/discussions/587_

### PR DESCRIPTION
**Pull request by puneeth072003**
### What kind of changes does this PR include?

-   [ ] Fixes or refactors
-   [ ] A new challenge
-   [x] Additional documentation
-   [ ] Something else

#### Description

Adding index as mentioned in the Issue #666  and discussion #587

### Relations

Closes #666

### Checklist:

-   [x] All the contributions made are solely the work of me and my co-authors
-   [x] I tested the changes in this PR (if applicable)
-   [ ] I added unit tests to ensure my change works (when change in Java or on front-end code)
-   [ ] The PR passes pre-commit hooks and automated tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,43 @@
 [![OpenSSF Best Practices](https://bestpractices.coreinfrastructure.org/projects/7024/badge)](https://bestpractices.coreinfrastructure.org/projects/7024)
 [![Discussions](https://img.shields.io/github/discussions/OWASP/wrongsecrets)](https://github.com/OWASP/wrongsecrets/discussions)
 
+### Table of contents
```
Comment (commjoen) [inline README.md:10]: Can you move the TOC to just above support please? So people can read what it is immediately:-)

**Review by commjoen — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (puneeth072003) [reply README.md:10]: above the heading?

Comment (commjoen) [reply README.md:10]: Yes please :-) above the heading

Comment (puneeth072003) [reply README.md:10]: Ok sir i'll do it in a jiffy....

**Review by commjoen — APPROVED** [review]
Thank you @puneeth072003 !

**Pull request merged**